From f51b15255a198552537aaa4774072d4901a4016a Mon Sep 17 00:00:00 2001
From: David Vrabel <david.vrabel@citrix.com>
Date: Mon, 10 Feb 2014 13:28:43 +0000
Subject: [PATCH] xen: install xen/gntdev.h and xen/gntalloc.h

xen/gntdev.h and xen/gntalloc.h both provide userspace ABIs so they
should be installed.

Signed-off-by: David Vrabel <david.vrabel@citrix.com>
---
 include/uapi/xen/Kbuild           |    2 ++
 include/{ => uapi}/xen/gntalloc.h |    0
 include/{ => uapi}/xen/gntdev.h   |    0
 3 files changed, 2 insertions(+), 0 deletions(-)
 rename include/{ => uapi}/xen/gntalloc.h (100%)
 rename include/{ => uapi}/xen/gntdev.h (100%)

diff --git a/include/uapi/xen/Kbuild b/include/uapi/xen/Kbuild
index 61257cb..5c45962 100644
--- a/include/uapi/xen/Kbuild
+++ b/include/uapi/xen/Kbuild
@@ -1,3 +1,5 @@
 # UAPI Header export list
 header-y += evtchn.h
+header-y += gntalloc.h
+header-y += gntdev.h
 header-y += privcmd.h
diff --git a/include/xen/gntalloc.h b/include/uapi/xen/gntalloc.h
similarity index 100%
rename from include/xen/gntalloc.h
rename to include/uapi/xen/gntalloc.h
diff --git a/include/xen/gntdev.h b/include/uapi/xen/gntdev.h
similarity index 100%
rename from include/xen/gntdev.h
rename to include/uapi/xen/gntdev.h
-- 
1.7.2.5

